application/controllers/Home.php

```diff
@@ -6,6 +6,21 @@ public function index()
         $this->load->model('Execution');
         $this->load->helper('my_duration');
 
+        //get all data from GCP
+        $GCP_files_list = [];
+        $url = getenv('QANB_GCPURL') !== false ? getenv('QANB_GCPURL') : 'https://storage.googleapis.com/prestashop-core-nightly/';
```

Comment:
```suggestion
        $url = getenv('QANB_GCPURL') ?: 'https://storage.googleapis.com/prestashop-core-nightly/';
```

Reply:
I genuinely didn't know this syntax, thanks